cmd/agent/main.go

```diff
@@ -17,19 +17,39 @@ package main
 import (
 	"fmt"
 	"os"
+	"time"
 
 	_ "github.com/joho/godotenv/autoload"
+	"github.com/rs/zerolog/log"
 	"github.com/urfave/cli/v2"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 
 	"github.com/woodpecker-ci/woodpecker/version"
 )
 
+const (
+	retryCount = 5
+	retryDelay = 2 * time.Second
+)
+
 func main() {
 	app := cli.NewApp()
 	app.Name = "woodpecker-agent"
 	app.Version = version.String()
 	app.Usage = "woodpecker agent"
-	app.Action = loop
+	app.Action = func(context *cli.Context) error {
```

Comment:
can we move that int it's own func and rename loop to something more descriptive ?